src/controllers/AuthController.js

```diff
@@ -242,4 +239,77 @@ class AuthController {
   }
 }
 
+/**
+ * Signup with their Crypto wallet
+ * 
+ * @author Eric Malaba
+ * @param {Object} req 
+ * @param {Object} res 
+ * @returns {Object} User
+ */
+export const walletSignup = async (req, res) => {
+  const { publicAddress } = req.body
+
+  const response = (status, user) => res.status(status).json({
+    nonce: user.nonce,
+    publicAddress: user.publicAddress,
+    updatedAt: user.updatedAt
+  });
+  const nonce = Math.floor(Math.random() * 1000000)
+  const user = await User
+    .findOneAndUpdate({ publicAddress }, { nonce }, { new: true })
```

Comment:
We need to create an organization here
` await Organization.create({
    user: user._id,
  });`

Reply:
> We need to create an organization here ` await Organization.create({ user: user._id, });`

Are you sure we should create organisation every time new user signup?